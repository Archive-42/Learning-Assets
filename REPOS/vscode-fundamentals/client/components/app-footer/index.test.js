// @ts-nocheck
import React from 'react';
import AppFooter from './index';
import renderer from 'react-test-renderer';

test('AppFooter renders as expected', () => {
  const component = renderer.create(
    <AppFooter></AppFooter>
  );
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});