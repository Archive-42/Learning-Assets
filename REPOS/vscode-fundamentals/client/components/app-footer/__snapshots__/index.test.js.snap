// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AppFooter renders as expected 1`] = `
<footer
  className="footer"
>
  <div
    className="mui-container-fluid"
  >
    ðŸ™Œ I built this in a 
    <a
      href="https://go/training-schedule"
      rel="noopener noreferrer"
      target="_blank"
    >
      LinkedIn
    </a>
     PWA course.
  </div>
</footer>
`;
